wordpress-mobile/WordPress-iOS
# Enable the feature flag for the new WordPress landing screen

Fixes #20057

To test:

1. Install / build WordPress app from this branch
2. Log out (if not already logged out)
3. Start app

Expect to see the revamped landing screen

## Regression Notes
1. Potential unintended areas of impact
Login

5. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and automated tests

6. What automated tests I added (or what prevented me from doing so)
Existing automated tests provide the needed coverage.


PR submission checklist:
